lib/bencher_json/src/project/measure/built_in.rs

```diff
@@ -115,4 +149,62 @@ pub mod iai_callgrind {
     create_measure!(AtTEndBlocks, "At t-end blocks", "at-t-end-blocks", "blocks");
     create_measure!(ReadsBytes, "Reads bytes", "reads-bytes", BYTES);
     create_measure!(WritesBytes, "Writes bytes", "writes-bytes", BYTES);
+
+    // Memcheck
+    create_measure!(MemcheckErrors, "Errors", "memcheck-errors", "errors");
```

Comment:
Yes, we should have different display names for the different `Errors` types.

I like your idea of separating the parse value from the final display name.

Preferably, you would create a new variant for the `create_measure!` macro that takes an additional argument, the display name.
So in the example here that would be `Memcheck Errors` or something like that.

The other macro variant would just call this longer form variant either with its parse value as the display name or with `None` (and have it be optional as you suggested).